src/marketo-munchkin/version.json

```diff
@@ -0,0 +1,3 @@
+{
+  "version" : "0.2.0"
```

Comment:
Just fyi this could be whatever we want since no tags have been published for this plugin yet (since it's new). But this version number is as good as any other so we'll roll with this.